src/components/Assets/AssetsList.tsx

```diff
@@ -377,7 +388,7 @@ const AssetsList = () => {
       <div className="mt-5 gap-3 space-y-2 lg:flex">
         <CountBlock
           text="Total Assets"
-          count={totalCount}
+          count={assetList?.count || 0}
```

Comment:
this is not correct. let's show loading state when data is not present yet. showing 0 while loading is showing misinformation

Reply:
@rithviknishad 
```.js
count={assetList?.count || 0}
loading={loading}
```
The loading prop is used to handle the loading state, and 0 is set as a fallback value to prevent displaying incorrect data. While loading, we show the loading state, not 0, so there’s no misinformation. 

This approach is consistent with other components like FacilityList, Facility Users, ManagePatients, and ManageUsers, where 0 is used as a fallback value while data is loading, and the correct value is shown once the data is available.